cardano-node/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -58,8 +57,8 @@ import           Cardano.Node.Protocol.Types
 -- This also serves a purpose as a sanity check that we have all the necessary
 -- type class instances available.
 --
-mkSomeConsensusProtocolCardano
-  :: NodeByronProtocolConfiguration
+mkSomeConsensusProtocolCardano ::
+     NodeByronProtocolConfiguration
```

Comment:
Why? https://github.com/input-output-hk/cardano-node/blob/master/STYLE.md#function-declaration

Reply:
In general, I agree, we should avoid making unnecessary changes, especially those which go against established local style.

Reply:
Done